backend/app/api/endpoints/knowledge.py

```
@@ -767,3 +789,173 @@ def get_qa_history(
+
+    summary_service = get_summary_service(db)
+    summary = await summary_service.get_document_summary(doc_id, current_user.id)
+    return DocumentSummaryResponse(document_id=doc_id, summary=summary)
+
+
+@summary_router.post("/{kb_id}/documents/{doc_id}/summary/refresh")
+async def refresh_document_summary(
+    kb_id: int,
+    doc_id: int,
+    background_tasks: BackgroundTasks,
+    current_user: User = Depends(security.get_current_user),
```

Ruff (0.14.10)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)